File: CS50/Week_6_Python/linear_search_6_15.py
import sys
# numbers = [4,6,2,8,1,0]
# for i in range(len(numbers)):
#     if numbeers[i] == 0:
#         print("found")


numbers = [4,6,2,8,1,0]
if 9 in numbers:
    print("found")
    sys.exit(0)

print("not found")
sys.exit(1)



import sys
names = ["ahmed" , "ail","gamal", "david" ,"khaled"]
if  'david' in names:
    print("Found")
    sys.exit(0)
print("not found")
sys.exit(1)

File: CS50/Week_6_Python/linear_search_6_15.c
#include <stdio.h>

int main(void){
    int numbers[] = {4,6,2,8,1,0};
    for(int i = 0 ; i < 6; i++){
        if(numbers[i] == 0){
            printf("Found/n");
            return 0;
        }
    }
    printf("Not Found/n");
    return 1;
}